src/TropicalGeometry/variety_supertype.jl

```diff
@@ -43,7 +43,7 @@ end
 @doc raw"""
     ambient_dim(TropV::TropicalVariety)
 
-Return the ambient dimension of `TropV`.  Requires `TropV` to be embedded.
+See [`ambient_dim(::PolyhedralComplex)`](@ref).
```

Comment:
I don't understand. In how far does the documentation for `ambient_dim(::PolyhedralComplex)` explain the behavior of `ambient_dim(TropV::TropicalVariety)` ? It feels to me as if there is some unspoken connection missing.

Reply:
Right.  The unspoken connection is that tropical varieties are weighted polyhedral complexes.

It is briefly mentioned at the beginning of the Tropical Varieties / Hypersurface / Linear Spaces / Curves sections, but I'm open to adding that bit of information to all constructor docstrings:
https://docs.oscar-system.org/stable/TropicalGeometry/variety/